contributors/design-proposals/encryption.md

```diff
@@ -0,0 +1,204 @@
+# Encryption
```

Comment:
Hi @andrewsykim, this is an interesting proposal. I'm doing some work to figure out the direction for secrets on the google side this quarter, it would be great to discuss this over VC with @smarterclayton and @jcbsmpsn.

I'm particularly interested in the use cases driving building this into kubernetes vs. taking advantage of the various external stores. Especially given there's a key management system for every cloud, and ones like vault that are already brokers for many different authn/z types and storage backends. If we could easily authenticate to a store that already handles key storage, encryption, rotation, and logging, would we still build it into kubernetes?